src/modules/taskList.js

```diff
@@ -0,0 +1,126 @@
+import Task from './task.js';
+import Status from './status.js';
+
+export default class TaskList {
+  constructor() {
+    this.listArray = [];
+  }
+
+  display() {
+    const listContainer = document.querySelector('.listContainer');
+    if (listContainer) {
+      listContainer.innerHTML = '';
+    }
```

Comment:
- [ ] [OPTIONAL] Great job here 👏. Your tasks are displayed when they are added however when I refresh the page, they disappear. When I check the local storage, I observed that they are still there. That means that It will be good if you improve the way you display them, so that when the page loads, they are displayed from **the local storage** 😃